spaceship/lib/spaceship/tunes/iap.rb

```diff
@@ -78,30 +88,20 @@ def create!(type: "consumable",
                            subscription_free_trial: subscription_free_trial)
 
         # Update pricing for a recurring subscription.
-        if type == Spaceship::Tunes::IAPType::RECURRING && pricing_intervals
+        if type == Spaceship::Tunes::IAPType::RECURRING &&
+           (pricing_intervals || subscription_price_target)
           # There are cases where the product that was just created is not immediately found,
           # and in order to update its pricing the purchase_id is needed. Therefore polling is done
           # for 4 times until it is found. If it's not found after 4 tries, a PotentialServerError
           # exception is raised.
           product = find_product_with_retries(product_id, 4)
```

Comment:
Can we bump this retry to maybe 6? (also update comments too). This seems to be a decently slow request on Apple's side 🙃

Reply:
Sure, does it work all the time with 6?